refinery/data_set_manager/tests.py

```diff
@@ -1572,3 +1571,62 @@ def test_get_basic_node(self):
         self.assertTrue('subanalysis' in self.get_response.data[0])
         self.assertTrue('type' in self.get_response.data[0])
         self.assertTrue('uuid' in self.get_response.data[0])
+
+
+class NodeIndexTests(APITestCase):
+
+    def setUp(self):
+        investigation = Investigation.objects.create()
+        study = Study.objects.create(investigation=investigation)
+        assay = Assay.objects.create(study=study)
+
+        test_file = StringIO()
+        test_file.write('Coffee is great.\n')
+        file_store_item = FileStoreItem.objects.create(
```

Comment:
I've found that something in our codebase is still writing temp files to disk for FileStoreItems regardless of the class you use for the `datafile` param

We have a pre_delete signal for FileStoreItems that deletes these datafiles, so I've been doing this in my tests:

```
def tearDown(self):
    FileStoreItem.objects.all().delete()
```